src/tests/nurse/test_views.py

```diff
@@ -54,39 +44,6 @@ def user2(db):
     user.delete()
 
 
-@pytest.fixture
-def staff_user(user):
-    """Creates and yields a staff user."""
-    user.is_staff = True
-    user.save()
-    yield user
-
-
-def authenticate_client_with_token(client, email, password):
-    """Authenticates a client using a token and returns it."""
-    # void previous credentials to avoid "Invalid token" errors
-    client.credentials()
-    response = client.post(
-        reverse_lazy("v2:api_token_auth"),
-        {"email": email, "password": password},
-        format="json",
-    )
-    assert response.status_code == status.HTTP_200_OK
-    token = response.data["token"]
-    client.credentials(HTTP_AUTHORIZATION=f"Token {token}")
-    return client
-
-
-@pytest.fixture
```

Comment:
Thanks for moving these up to conftest.py as this PR gets ready we will split and create a dedicated one for refactoring changes so the Stripe PR stay lean and easy to follow

Reply:
https://github.com/mynotif/mynotif-backend/pull/212